num_enum_derive/src/lib.rs

```diff
@@ -278,6 +305,14 @@ impl Parse for EnumInfo {
                         }
                         attr_spans.default.push(attribute.span());
                         is_default = true;
+                    } else if attribute.path.is_ident("catch_all") {
```

Comment:
Let's remove bare `#[catch_all]` support and only support `#[num_enum(catch_all)]` - we only added `#[default]` support because Rust itself started supporting `#[default]` as a built-in attribute, but generally scoping attributes is good :)